src/CBCNodeListViewController.m

```diff
@@ -206,6 +206,9 @@ static void CBCAddNodeFromBreadCrumbs(CBCNode *tree,
     CBCNode *child = [[CBCNode alloc] initWithTitle:title];
     [node addChild:child];
     child.metadata = metadata;
+    if ([[child.metadata objectForKey:CBCIsPrimaryDemo] boolValue]) {
```

Comment:
You may have to explicitly test against NO or !NO to avoid a static analyzer warning.

Reply:
thanks :)